returns/functions.py

```diff
@@ -144,3 +148,12 @@ def raise_exception(exception: Exception) -> NoReturn:
 
     """
     raise exception
+
+
+def not_(
```

Comment:
I would say that `not_` has different type.

It is more like: `def not_(function: Callable[..., bool]) -> Callable[..., bool]:`
This function can take any args and kwargs from the callee function.

We can tweak it in `mypy` plugin to work correctly with type (currently it is required).

Reply:
The implementation will be like this:
```python
def not_(not_(function: Callable[..., bool]) -> Callable[..., bool]:
    return lambda *args, **kwargs: not function(*args, **kwargs)
```
right?